src/IndProp.lidr

```diff
@@ -162,7 +167,7 @@ Let's look at a few examples to see what this means in practice.
 
 === Pattern Matching on Evidence
 
-\todo[inline]{Edit the whole section to talk about dependent pattern matching}
+\ \todo[inline]{Edit the whole section to talk about dependent pattern matching}
```

Comment:
The first one is a literal space, `\ `, and the second is for `\todo`. This way it doesn't try to put the note in line with the heading (which almost always falls off the page).